cmd/create/kubeletconfig/cmd_test.go

```diff
@@ -124,30 +124,6 @@ var _ = Describe("create kubeletconfig", func() {
 				ContainSubstring("Failed getting KubeletConfig for cluster 'cluster'"))
 		})
 
```

Comment:
This test had to be deleted as we now force the user into interactive mode and we have no good way of testing interactive mode (yet).